models/commons/src/discounted-price/discounted-price-draft/index.ts

```diff
@@ -0,0 +1,9 @@
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+
+export const DiscountedPriceDraftRest = {
+  random: RestModelBuilder,
+};
+
+export const DiscountedPriceDraftGraphql = {
+  random: GraphqlModelBuilder,
+};
```

Comment:
I'm missing the `presets` prop in both models ([reference](https://github.com/commercetools/test-data/blob/main/generators/src/new-test-model/templates/model/index.tpl#L7)).

Reply:
thanks Addressed here [3eec618](https://github.com/commercetools/test-data/pull/698/commits/3eec618f50b5e798b19c2da0ad3ebd81c30c05c6)